test/e2e/metamask-ui.spec.js

```diff
@@ -200,9 +200,10 @@ describe('MetaMask', function () {
       await restoreSeedLink.click();
       await driver.delay(regularDelayMs);
 
-      await driver.clickElement('.import-account__checkbox-container');
-
-      await driver.fill('.import-account__secret-phrase', testSeedPhrase);
+      await driver.fill(
+        'input[placeholder="Enter your Secret Recovery Phrase"]',
```

Comment:
I couldn't see a way to pass a `data-testid` prop to our TextField component, so instead I'm using a strategy already employed in our e2e tests elsewhere to select using the placeholder text instead.

Reply:
This comment is outdated; this event is now preserved in the `CreateNewVault` component. It's useless but still better to keep this PR focused. Plus it is present in some reports in Mixpanel as well.